terminfo/mkinfo.go

```diff
@@ -225,6 +225,11 @@ func getinfo(name string) (*terminfo.Terminfo, string, error) {
 			return nil, "", err
 		}
 	}
+
+	if !addTrueColor && tc.getstr("Tc") != "" {
```

Comment:
The first part of this check is redundant/pointless, since we just set it after.

Reply:
I like the idea of using this terminfo capability, and I recall seeing it somewhere, but I can't find any terminfo entries that carry it.